npm install fails for package consumers due to "only-allow pnpm"

### Describe the bug

First line of the readme Installation section suggests you can use npm, yarn, or pnpm to install faker:

`npm install @faker-js/faker --save-dev`

```
❯ npm install @faker-js/faker --save-dev
npm ERR! code 1
npm ERR! path ~/dev/myproject/node_modules/@faker-js/faker
npm ERR! command failed
npm ERR! command sh -c npx only-allow pnpm
npm ERR! ╔═════════════════════════════════════════════════════════════╗
npm ERR! ║                                                             ║
npm ERR! ║   Use "pnpm install" for installation in this project.      ║
npm ERR! ║                                                             ║
npm ERR! ║   If you don't have pnpm, install it via "npm i -g pnpm".   ║
npm ERR! ║   For more details, go to https://pnpm.js.org/              ║
npm ERR! ║                                                             ║
npm ERR! ╚═════════════════════════════════════════════════════════════╝

npm ERR! A complete log of this run can be found in:
npm ERR!     ~/.npm/_logs/2022-01-24T23_47_37_989Z-debug.log
```

This commit (https://github.com/faker-js/faker/commit/a273f67066c7c19b01ff4348eb9d7e857615cca6) added `"preinstall": "npx only-allow pnpm"` to `package.json`. 

### Reproduction

`npm install @faker-js/faker --save-dev`

### Additional Info

_No response_

The same happens to me.

```
⚡️ npm install @faker-js/faker --save-dev
npm ERR! code 1
npm ERR! path /Users/edilson/Desktop/ze-projects/ze-communication-service/node_modules/@faker-js/faker
npm ERR! command failed
npm ERR! command sh -c npx only-allow pnpm
npm ERR! ╔═════════════════════════════════════════════════════════════╗
npm ERR! ║                                                             ║
npm ERR! ║   Use "pnpm install" for installation in this project.      ║
npm ERR! ║                                                             ║
npm ERR! ║   If you don't have pnpm, install it via "npm i -g pnpm".   ║
npm ERR! ║   For more details, go to https://pnpm.js.org/              ║
npm ERR! ║                                                             ║
npm ERR! ╚═════════════════════════════════════════════════════════════╝

npm ERR! A complete log of this run can be found in:
npm ERR!     ~/.npm/_logs/2022-01-25T00_21_50_304Z-debug.log
```

--- a/package.json
+++ b/package.json
@@ -51,7 +51,6 @@
     "tsconfig.json"
   ],
   "scripts": {
-    "preinstall": "npx only-allow pnpm",
     "build:clean": "rimraf dist",
     "build:code": "esno ./scripts/bundle.ts",
     "build:types": "tsc --emitDeclarationOnly --outDir dist/types",

